docs/pages/reference/js/getFrame.mdx

```diff
@@ -1,25 +1,54 @@
 # getFrame
 
-Returns a `Frame` object, extracting the frame metadata from the given htmlString, or returning null if the frame is invalid. Includes tags from the [Open Frames specification](https://github.com/open-frames/standard).
+Parses HTML according to a given frames specification and returns an object containing a [`Frame`](reference/js/types#reference) if it is valid. Supported specifications are `farcaster` and [`openframes`](https://github.com/open-frames/standard) - defaults to `farcaster` if no specification is specified.
 
```

Comment:
can we add a `Parameters` section here similar to https://framesjs.org/reference/core/createFrames#parameters

and a description of the return value